rest-server/src/main/resources/nubesgen/.github/workflows/gitops.yml.mustache

```diff
@@ -86,6 +86,44 @@ jobs:
           application_name: ${{ steps.infrastructure-deployment.outputs.application_name }}
           container_registry_name:  ${{ steps.infrastructure-deployment.outputs.container_registry_name }}
     <%/runtimeQuarkus%>
+    <%#runtimeMicronaut%>
+      <%#runtimeMaven%>
+      - name: Build a Micronaut docker image project using Maven
+        uses: microsoft/nubesgen-actions/gitops-build-java-maven@<%compositeActionsVersion%>
+        with:
+          build_command: mvn clean package -Dpackaging=docker -Djib.to.image=${{ steps.infrastructure-deployment.outputs.container_registry_name }}.azurecr.io/${{ steps.infrastructure-deployment.outputs.application_name }}/${{ steps.infrastructure-deployment.outputs.application_name }}:${{ github.sha }}
+      <%/runtimeMaven%>
+      <%#runtimeGradle%>
+      - name: Build a Micronaut docker image project using Gradle
+        uses: microsoft/nubesgen-actions/gitops-build-java-gradle@<%compositeActionsVersion%>
+        id: dockerbuild
+        with:
+          build_command: gradle dockerBuild | tee stdout.txt
```

Comment:
should this use the Gradle Wrapper instead?

Reply:
Done